feelpp/cmake/modules/feelpp.macros.cmake

```diff
@@ -394,7 +394,7 @@ macro(feelpp_add_test)
            add_test(NAME feelpp_test_${FEELPP_TEST_NAME}-np-${NProcs2} COMMAND /bin/sh -c "exit 77")
            set_tests_properties( feelpp_test_${FEELPP_TEST_NAME}-np-${NProcs2} PROPERTIES SKIP_RETURN_CODE 77 )
         else()
-          add_test(NAME feelpp_test_${FEELPP_TEST_NAME}-np-${NProcs2} COMMAND ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} ${NProcs2} ${MPIEXEC_PREFLAGS} ${CMAKE_CURRENT_BINARY_DIR}/${targetname} --log_level=test_suite ${BOOST_TEST_SEPARATOR} ${MPIEXEC_POSTFLAGS} ${FEELPP_TEST_CFG_CLI} ${FEELPP_TEST_CLI} --directory=testsuite/test_${FEELPP_TEST_NAME} --rm )
+          add_test(NAME feelpp_test_${FEELPP_TEST_NAME}-np-${NProcs2} COMMAND ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} ${NProcs2} ${MPIEXEC_PREFLAGS} ${CMAKE_CURRENT_BINARY_DIR}/${targetname} --log_level=test_suite ${BOOST_TEST_SEPARATOR} ${MPIEXEC_POSTFLAGS} ${FEELPP_TEST_CFG_CLI} ${FEELPP_TEST_CLI} --directory=testsuite/test_${FEELPP_TEST_NAME} --repository.append.date=false --rm )
```

Comment:
it is really necessary? not by default?

Reply:
not quite the user may configure appending dates in the `.feelppconfig`.
since we `--rm` it may not be that important anyway